source/nlopt.d

```diff
@@ -163,7 +163,7 @@ void nlopt_srand(ulong seed);
 void nlopt_srand_time();
 void nlopt_version(int* major, int* minor, int* bugfix);
 
-struct nlopt_opt_s{}; /* opaque structure, defined internally */
+struct nlopt_opt_s; /* opaque structure, defined internally */
```

Comment:
Using a truly opaque struct means that it is a compile-time error for someone to try to use one directly (e.g. `nlopt_opt_s myopt;`), which is nice.